src/main/java/net/snowflake/ingest/connection/TelemetryService.java

```diff
@@ -73,7 +81,9 @@ public void close() {
     this.telemetry.close();
   }
 
-  /** Report the Streaming Ingest latency metrics */
+  /** Reports aggregated the Streaming Ingest latency metrics */
+  // TODO @rcheng question - can I rename the telemetry method and type to
```

Comment:
I think it would be better to have the following, however I'm not sure if renaming telemetry types/names would break anything downstream

- reportAggregatedLatencyInSec
- reportBlobLatencyInSec